src/siteconfig/models.py

```diff
@@ -1,18 +1,42 @@
 from django.contrib.auth import get_user_model
 from django.core.cache import cache
 from django.core.exceptions import MultipleObjectsReturned
-from django.db import connection, models
+from django.db import connection, models, transaction
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 from django.shortcuts import get_object_or_404
 from django.templatetags.static import static
+from django.conf import settings
 
 from django_tenants.utils import get_public_schema_name
 from redis import exceptions as redis_exceptions
 
 User = get_user_model()
 
 
+def get_default_deck_owner():
+    """ 
+        This is run once during site initialization
+
+        Also need to initialize owner if none exists yet.
+        (Since initialization.py loads after everything else)
+    """ 
+    qs = User.objects.filter(username=settings.TENANT_DEFAULT_OWNER_USERNAME)
```

Comment:
Don't use `qs`.  give variables meaningful names. `owner_user_qs` or something like that

Reply:
done